app/models/user.rb

```diff
@@ -47,7 +47,11 @@ def by_conference(conference)
   has_many :event_users, dependent: :destroy
   has_many :events, -> { uniq }, through: :event_users
   has_many :presented_events, -> { joins(:event_users).where(event_users: {event_role: 'speaker'}).uniq }, through: :event_users, source: :event
-  has_many :registrations, dependent: :destroy
+  has_many :registrations, dependent: :destroy do
+    def for_conference conference
+      where(conference: conference)
```

Comment:
a user should have more that one registration for a conference right? So may make sense to only return one registration here

Reply:
You mean, that only can have one, right? that's why returning only one registration.

Reply:
exactly

Reply:
👍